<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeDialog</class>
 <widget class="QDialog" name="AttributeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Attribute Extraction</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/ui/icons/vex_logo.png</normaloff>:/ui/icons/vex_logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QWidget" name="fileWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>CSV File</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLineEdit" name="leCSVFilePath">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pbCSVFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="localHistogramWidget">
     <property name="title">
      <string>Local histogram:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="lblSide">
        <property name="text">
         <string>Texture dim (n×n×n):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbSide">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="singleStep">
         <number>2</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="bottomVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="topVerticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="featureWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="pbSelectAll">
        <property name="text">
         <string>Select all</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBoxTypes">
        <property name="title">
         <string>Attributes</string>
        </property>
        <layout class="QGridLayout" name="groupBoxTypesLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="bottomWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QPushButton" name="pbApply">
        <property name="text">
         <string>OK</string>
        </property>
        <property name="shortcut">
         <string>Enter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pbCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbSelectAll</tabstop>
  <tabstop>sbSide</tabstop>
  <tabstop>pbApply</tabstop>
  <tabstop>pbCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbApply</sender>
   <signal>clicked()</signal>
   <receiver>AttributeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>AttributeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotApply()</slot>
  <slot>slotCancel()</slot>
 </slots>
</ui>
